src/app/components/Home/Home.jsx

```diff
@@ -15,9 +15,11 @@ const Home = ({ sortBy }) => (
 
     <div className="content-wrapper">
       <div className="nypl-column-three-quarters nypl-column-offset-one">
```

Comment:
do not remove the `content-wrapper`. it should really be `nypl-full-width-wrapper` and the general structure would be:

````html
<div class="nypl-full-width-wrapper">
  <div class="nypl-row">
    <div class="nypl-column-three-quarters nypl-column-offset-one">

      <p class="nypl-lead">…</p>

      <fieldset class="nypl-omnisearch">
      …
      </fieldset>

    </div>
  </div>
</div>
````